### ISSUE TITLE
MiDaS 2.1

### ISSUE THREAD

**Issue by 3dsf** [yan99033/MiDaS-cpp#5]
Just to confirm, this is compatible with **MiDaS <= 2.1**

https://github.com/intel-isl/MiDaS/releases/tag/v2_1

**Comment by yan99033** [comment]
You are right. To use MiDaS 3.0, you just have to go to `python/export_model.py` and import the `DPTDepthModel` class to convert the pre-trained model to a torchscript model.

**Comment by hhxxzxy** [comment]
> 
Could you please teach me the detailed steps?thank you
